CONTRIBUTING_DOCS.md

```diff
@@ -0,0 +1,211 @@
+
+## More Complex Changes / Building the PHP documentation
+
+To build and view the documentation after making more extensive changes
+(e.g. adding entire sections or files), you will need to
+[clone some of the tooling](#set-up-a-development-environment)
+(doc-base, phd and possibly web-php)
+in addition to the language repository you want to
+[make the changes](#make-changes-to-the-documentation) to.
+You need to [validate the changes](#validate-the-documentation)
+and you can (but do not have to) [render the documentation](#render-the-documentation)
+before you [open a PR](#commit-changes-and-open-a-pr).
+
+If you'd like to know more about what each repository is
+and/or how PHP's documentation is built please refer to
+this [overview](https://github.com/php/doc-base/OVERVIEW.md).
+
+### Requirements
+
+Following this guide to make changes to and build the documentation has the following dependencies:
+ - PHP 8.0+ with the DOM, libXML2, XMLReader and SQLite3 extensions
+ - Git
+ - an IDE or a text editor
+
+### Set up a development environment
+
+#### Basic setup
+To start working on the English documentation, you need to fork it on Github
+and clone it, doc-base (for validating the XML files) and PhD (for rendering the files).
+
+  ```shell
+  git clone https://github.com/<your_github_username>/<your_fork> en
+  git clone https://github.com/php/doc-base
+  git clone https://github.com/php/phd
+  cd en
+  git remote add upstream https://github.com/php/doc-en
+  ```
+where `<your_github_username>` and `<your_fork>` needs to be replaced by
+your GitHub username and the name of your for of the English documentation respectively.
+Please note that the English documentation has to be cloned into the `en` directory.
+
+To clone any of the translations, replace `doc-en en` at the end of the first line
+with the translation you would like to clone
+(e.g. `doc-pt_br` for the Brazilian documentation).
+As an example, the following shows how to set up the Brazilian documentation:
+
+  ```shell
+  git clone https://github.com/<your_github_username>/doc-pt_br
+  git clone https://github.com/php/doc-base
+  git clone https://github.com/php/phd
+  cd doc-pt_br
+  git remote add upstream https://github.com/php/doc-pt_br
+  ```
+where `<your_github_username>` needs to be replaced by your GitHub username.
+
+<details>
+  <summary>List of languages/repositories</summary>
+
+  - [Brazilian Portugues](https://github.com/php/doc-pt_br) (doc-pt_br)
+  - [Chinese(Simplified)](https://github.com/php/doc-zh) (doc-zh)
+  - [English](https://github.com/php/doc-en) (doc-en)
+  - [French](https://github.com/php/doc-fr) (doc-fr)
+  - [German](https://github.com/php/doc-de) (doc-de)
+  - [Italian](https://github.com/php/doc-it) (doc-it)
+  - [Japanese](https://github.com/php/doc-ja) (doc-ja)
+  - [Russian](https://github.com/php/doc-ru) (doc-ru)
+  - [Spanish](https://github.com/php/doc-es) (doc-es)
+  - [Turkish](https://github.com/php/doc-tr) (doc-tr)
```

Comment:
Polish is missing :)

Reply:
I deliberately didn't add languages/repositories that aren't available through php.net (Polish, Romanian and Ukrainian) but since there is a repo for each of these it could kind of make sense to add them. I leave the decision up to you. :-)

Reply:
I think it makes sense to add them, the Polish one is being active again and builds, Ukrainian has a maintainer, Romanian is the only one that is in limbo AFAIK.

Reply:
I've added all three of them.